docs/channel_considerations.mdx

```diff
@@ -0,0 +1,48 @@
+# Channel considerations and specificities
+
+While the life cycle categories (production, distribution, consumer use and disposal) remain the same for all advertising channels, each channel presents specificities that need to be taken into account for accurate emissions modelling.
+For instance: static out-of-home billboards do not have content, just the ad, but they will have a disposal component when the vinyl is removed or recycled.
+
+The below sections describe the considerations and specificities taken into account to model different channels.
+
+## Digital out-of-home (DOOH)
+
+### Boundaries of measurement
+
+DOOH screens, like consumer devices, require energy to display an ad. Energy consumption during the use phase is in fact the highest source of emissions.
+However, on the contrary to consumer devices, billboards are owned or leased by media owners who therefore have a high level of operational control over the screen attributes that influence embodied and energy related emissions (venue category, manufacturer, model, size, location and positioning, operating hours, brightness settings etc...). For that reason, we are including emissions from screens' energy usage and production to "media distribution".
+
+### Emissions from screens' energy consumption
+
+On the contrary to [consumer device emissions](./consumer_devices.mdx), we have found no credible research paper describing the power draw of large digital billboards. As such, we estimate for individual screens based on a variety of input data sources (listed in order from most accurate to least accurate):
+
+- Electricity metres installed on the screens themselves and monitoring power draw in near real-time.
+- Power bills received by media owners on a monthly, quaterly or yearly basis, for a group of or for individual screens.
+- Technical specifications from screen manufacturers like Daktronics, Samsung or Panasonic. We have found that these often document maximum power draw (instead of average power draw), which can lead to an over-estimation of energy consumption.
+- Extrapolation based on physical dimensions and one of the above data point for similar screens.
+- Extrapolation based on physical dimensions with no other data, off the back of data kindly provided to us by early partners.
```

Comment:
should we have some example math here? or reference where we're getting these defaults?